### PR TITLE
LINK-1982, LINK-1984 | Admin page permissions

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
- Display only read-only fields for keywords and keywords sets and prevent to modify them. 
- Add feature flag to enable place admin pages. Default value is false
- Allow organization admin pages only for financial admins and superusers
- Allow price group admin pages only for financial admins and superusers
- Allow only admin users to see Linked Registration instructions

## Closes
[LINK-1982](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1982)
[LINK-1984](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1984)

[LINK-1982]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1982?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LINK-1984]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1984?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr386-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr386-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr386-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://linkedcomponents-pr386-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr386-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://linkedcomponents-pr386-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr386-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr386-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr386-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: src/domain/admin/layout/AdminPageLayout.tsx [hunk 1]
```diff
@@ -18,9 +22,15 @@ const AdminPageLayout: React.FC<PropsWithChildren> = ({ children }) => {
     { label: t('keywordsPage.title'), to: ROUTES.KEYWORDS },
     { label: t('keywordSetsPage.title'), to: ROUTES.KEYWORD_SETS },
     { label: t('imagesPage.title'), to: ROUTES.IMAGES },
-    { label: t('organizationsPage.title'), to: ROUTES.ORGANIZATIONS },
-    { label: t('placesPage.title'), to: ROUTES.PLACES },
-    { label: t('priceGroupsPage.title'), to: ROUTES.PRICE_GROUPS },
+    ...(arePriceGroupRoutesAllowed(user)
```
Comment (harriris-vincit) [inline src/domain/admin/layout/AdminPageLayout.tsx:25]: Although these are currently allowed for the same users, it is a tiny bit confusing when taking a first look at this and seeing that price group permissions are checked for organization routes. Creating a dedicated function for organization route permissions check or renaming this to something like `areFinancialRoutesAllowed` could make this easier to read.

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr386-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr386-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: src/domain/admin/layout/AdminPageLayout.tsx (hunk 1 above)
Comment (jorilindell) [reply src/domain/admin/layout/AdminPageLayout.tsx]: Renamed the function as `areFinancialRoutesAllowed`. Also added check for all the organization routes so user without sufficient permissions is not allowed to see the organization pages

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr386-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Review by harriris-vincit — APPROVED** [review]
LGTM

**Pull request merged**